frontend/src/App.js

```diff
@@ -1,16 +1,22 @@
 import React, { Component } from "react";
 import Booklist from "./containers/Booklist";
-// import "./App.css";
+import Description from "./components/Description";
+
+import "./App.css";
+import { BrowserRouter as Router, Route } from "react-router-dom";
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <h1 className="App-title">My Book List</h1>
-        </header>
-        <Booklist/>
-      </div>
+      <Router>
+        <div className="App">
+          <header className="App-header">
+            <h1 className="App-title">My Book List</h1>
+          </header>
+          <Route exact path="/" component={Booklist} />
+          <Route path="/description" component={Description} />
```

Comment:
If you add the following at the bottom then any bad routes will render the main page
```
<Route path="/description" component={Booklist} />
```

Reply:
added:
<Route path="/" component={Booklist} />